owncloud/ios-app
# [feature/improve-public-link-creation] Improve creation of public link

## Description
- enable dragging public link
- added clear button on public link name
- copy link to clipboard, after link was created

## Related Issue
#671 

## Motivation and Context
Better interaction for public links

## How Has This Been Tested?
- on iPad drag a created public link outside of the app
- create a public link and pasteboard should contain the new link by default
- check if public link name has a clear button inside

## Types of changes


## Checklist:



## QA

- [ ] (1) Notify link copied to clipboard https://github.com/owncloud/ios-app/pull/673#issuecomment-617839336
